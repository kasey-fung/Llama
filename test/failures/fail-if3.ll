int main(int x, int y):
    int a 
    int b 
    int c 
    int d 

    if (a < b):
        c = d
    else:
        sum()
