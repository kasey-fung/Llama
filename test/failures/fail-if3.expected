Fatal error: exception Failure("unrecognized function sum")